app/code/Magento/Customer/i18n/en_US.csv

```diff
@@ -541,3 +541,4 @@ Addresses,Addresses
 "The Date of Birth should not be greater than today.","The Date of Birth should not be greater than today."
 "The store view is not in the associated website.","The store view is not in the associated website."
 "The Store View selected for sending Welcome email from is not related to the customer's associated website.","The Store View selected for sending Welcome email from is not related to the customer's associated website."
+"The group id specified is invalid. Please specify valid group id.","The group id specified is invalid. Please specify valid group id."
```

Comment:
Please, modify the error message according my comments above. Thank you!

Reply:
@rogyar Modified.